lib/stronger_parameters/constraints/ulid_constraint.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'stronger_parameters/constraint'
+
+module StrongerParameters
+  class UlidConstraint < Constraint
+    # https://www.crockford.com/base32.html
+    INVALID_CHAR_REGEX = /[ilou]|[^a-z0-9]/i.freeze
```

Comment:
The base32-crockford gem does have a method to [validate](https://github.com/levinalex/base32/blob/master/lib/base32/crockford.rb#L124) character sets, and ulid-rails defers to that to [check validity](https://github.com/k2nr/ulid-rails/blob/master/lib/ulid/rails/validator.rb#L8). It seemed better to avoid pulling in base32-crockford as a dependency here though.